rule-audit-client/src/redux/modules/capiContent/index.ts

```diff
@@ -1,82 +1,15 @@
-import { Dispatch } from "redux";
-import set from "lodash/fp/set";
-import AppTypes from "AppTypes";
 import { createAsyncResourceBundle } from "redux-bundle-creator";
 
-import { CapiContentWithMatches, fetchCapiSearch } from "services/capi";
-import { getBlocksFromHtmlString } from "utils/prosemirror";
-import { fetchTyperighterMatches } from "services/typerighter";
-import { notEmpty } from "utils/predicates";
+import { CapiContentWithMatches } from "services/capi";
+import * as _selectors from './selectors';
 
-const bundle = createAsyncResourceBundle<CapiContentWithMatches, {}, "capi">(
+export * as thunks from './thunks';
+
+export const { actions, reducer, ...bundle } = createAsyncResourceBundle<CapiContentWithMatches, {}, "capi">(
```

Comment:
I've moved the thunks into their own file for convenience sake, as this file was getting a little large.